controllers/spacebindingrequestmigration/spacebindingrequest_migration_controller.go

```diff
@@ -0,0 +1,183 @@
+package spacebindingrequestmigration
+
+import (
+	"context"
+	"fmt"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/codeready-toolchain/host-operator/pkg/cluster"
+	errs "github.com/pkg/errors"
+	"github.com/redhat-cop/operator-utils/pkg/util"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+// Reconciler reconciles a SpaceBindingRequestMigration object
+type Reconciler struct {
+	Client         runtimeclient.Client
+	Scheme         *runtime.Scheme
+	Namespace      string
+	MemberClusters map[string]cluster.Cluster
+}
+
+// SetupWithManager sets up the controller with the Manager.
+func (r *Reconciler) SetupWithManager(mgr ctrl.Manager, memberClusters map[string]cluster.Cluster) error {
+	// since it's mandatory to add a primary resource when creating a new controller,
+	// we add the SpaceBindingRequest CR even if there should be no reconciles triggered from the host cluster,
+	// only from member clusters (see watches below)
+	// SpaceBindingRequest owns spacebindings so events will be triggered for those from the host cluster.
+	b := ctrl.NewControllerManagedBy(mgr).
+		For(&toolchainv1alpha1.SpaceBindingRequest{}).
```

Comment:
Shouldn't it watch `SpaceBindings` in the host? And not `SBRs` in members?

Reply:
> Shouldn't it watch SpaceBindings in the host? 

I've repaced the `For` resource to use SpaceBindings, see https://github.com/codeready-toolchain/host-operator/pull/922/commits/038c6142d80ea7cd0d38f212c1e3b9d9a7f55438

> And not SBRs in members?

I thought that watching for SBR changes would be good, in case something changes the migration controller will reconcile it. Worst case it should not harm since the spacebinding will have an SBR already and will exit here https://github.com/codeready-toolchain/host-operator/pull/922/files#diff-d49007122db69f04ba9af1be1154a194e84eaaef10df1cb1caa211e54835a63bR76-R78

Also IIRC we need both a primary (`For`) and a secondary resource (`Watches`) , but I might be wrong.

Reply:
OK. The For() and Watches look good now since we actually have SBs as the primary resources and SBRs as secondary for this controller. However the description is still incorrect. I will create a new comment for that.